index.md

```diff
@@ -45,10 +45,11 @@ A practice is included in this minimal list if:
 2.  All syntheiszed data is stored in well-defined widely-used formats:
     *   CSV or HDF5 for tabular data
     *   JSON or YAML for referential data
-3.  All data is normalized:
-    *   Atomic values
-    *   Keys to identify and correct records
-4.  Normalization is treated as a processing step
+3.  All data follows a few basic rules:
+    *   Each value is *atomic*, i.e., has no sub-parts
+        *   Example: store personal and family names in separate fields
+    *   Every record has a unique *key* so that it can be selected precisely
+4.  *Normalization* (the process of putting data into this form) is treated as a processing step
```

Comment:
'into this form' may be confusing. How about 'the process of modifying data to adhere to the rules of point 3'?